gtk4/src/subclass/application.rs

```diff
@@ -19,12 +18,12 @@ pub trait GtkApplicationImpl:
     }
 }
 
-pub trait GtkApplicationImplExt {
+pub trait ApplicationImplExt {
```

Comment:
Why did you rename the trait? This is annoying now because `gio::ApplicationImplExt` and `gtk::ApplicationImplExt` :)

Reply:
I've followed what's done in gtk3 here

Reply:
Ah no. I don't know what I was doing here. Fixed!